snuba/snuba_migrations/metrics/0035_metrics_raw_timeseries_id.py

```diff
@@ -0,0 +1,48 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.ClickhouseNodeMigrationLegacy):
```

Comment:
Lets not use the legacy migration. It is meant to be [deprecated](https://github.com/getsentry/snuba/blob/master/snuba/migrations/migration.py#L186-L189). Use the newer class instead https://github.com/getsentry/snuba/blob/master/snuba/migrations/migration.py#L97.

Reply:
I wonder if the legacy migration could log a warning or something (don't think it should block migrating).

Though I guess being named `Legacy` is already nudging you to not use it

Reply:
Thanks for pointing this out, I hadn't noticed this.